tasks/main.yaml

```diff
@@ -0,0 +1,100 @@
+---
+- name: Install required pakages
+  package:
+    name: "{{ item }}"
```

Comment:
Set a custom loop variable name.

Reply:
`item` is a common thing in Ansible. While it is possible to rename it pre 2.7, the output might be buggy. Therefore I will leave it like that.